scripts/deploy.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+
+# Exit in case of error
+# make sure that you are in `scripts` dir
+set -e
+
+path=../tmp
+file=git_version
+dest_dir="${path}/${file}"
+if [ ! -d $path ] ; then
+  mkdir $path
+  echo "${path} path created."
+fi
+
+chmod +x ./git_version_for_docker.sh
+./git_version_for_docker.sh > $dest_dir
+
+cd ..
+
+make build-image
+make run-database
+make run-docker
```

Comment:
Still, this script is not used anywhere, how should our deployment make use of this?